python/vyos/ifconfig/interface.py

```diff
@@ -898,6 +899,82 @@ def add_to_bridge(self, bridge_dict):
             # set bridge port path priority
             if 'priority' in bridge_config:
                 self.set_path_cost(bridge_config['priority'])
+            
+            vlan_filter = 0
+            
+            vlan_del = set()
+            vlan_add = set()
+
+            if 'native_vlan' in bridge_config:
+                vlan_filter = 1
+                cmd = f'bridge vlan del dev {self.ifname} vid 1'
+                self._cmd(cmd)
+                vlan_id = bridge_config['native_vlan']
+                if int(vlan_id) != 1:
+                    vlan_del.add(1)
+                cmd = f'bridge vlan add dev {self.ifname} vid {vlan_id} pvid untagged master'
+                self._cmd(cmd)
+                vlan_add.add(vlan_id)
+
+            if 'allowed_vlan' in bridge_config:
+                vlan_filter = 1
+
+            if vlan_filter:
+                if 'native_vlan' not in bridge_config:
+                    cmd = f'bridge vlan del dev {self.ifname} vid 1'
+                    self._cmd(cmd)
+
+            if 'allowed_vlan' in bridge_config:
+                for vlan in bridge_config['allowed_vlan']:
+                    cmd = f'bridge vlan add dev {self.ifname} vid {vlan} master'
+                    self._cmd(cmd)
+                    vlan_add.add(vlan)
+            
+            vlan_bridge_ids = Section.klass(bridge)(bridge, create=True).get_vlan_ids()
+            
+            # Delete VLAN ID for the bridge
+            for vlan in vlan_del:
+                if isinstance(vlan,str) and vlan.isnumeric():
+                    if int(vlan) == 1 and int(vlan) not in vlan_bridge_ids:
+                        cmd = f'bridge vlan del dev {bridge} vid {vlan} self'
```

Comment:
Why is this line required four times? This is too redundant. It must be shrinked to one occurance.

![image](https://user-images.githubusercontent.com/25299219/101677412-d6cb9300-3a5c-11eb-8bc1-ffc78527d640.png)